doc/api/cli.md

```diff
@@ -933,6 +933,15 @@ files with no extension will be treated as WebAssembly if they begin with the
 WebAssembly magic number (`\0asm`); otherwise they will be treated as ES module
 JavaScript.
 
+### `--experimental-enable-transformation`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+Enables the transformation of TypeScript-only syntax into JavaScript code.
+Implies `--experimental-strip-types`.
```

Comment:
I think it would be good to list the differences between the two flags, like the following features will be enabled and no others extra (like [import elision](https://www.typescriptlang.org/tsconfig/#verbatimModuleSyntax), or experimental decorators):

* `Enum`
* `namespaces` and legacy `module` keyword namespace (inner module)
* parameter properties

Debatably, if the legacy `module` keyword is supported (the PR currently does), we'll need to deprecate it regarding https://github.com/microsoft/TypeScript/issues/51825 and https://github.com/tc39/proposal-module-declarations on the way.

Reply:
Ill add documentation

Reply:
added